src/main/res/values-de/strings_pref.xml

```diff
@@ -78,7 +78,7 @@
     <string name="pref_sensors_orient_filter_medium">Mittlerer Filter</string>
     <string name="pref_sensors_orient_filter_no_filter">Kein Filter</string>
     <string name="pref_statusbar">Statusbar-Symbol</string>
-    <string name="pref_statusbar_desc">Symbol in der Statusbar anzeigen</string>
+    <string name="pref_statusbar_desc">Symbol in der Statusbar anzeigen (Wen diese Option nicht verfügbar ist, wird die Statusbar durch eine andere Einstellung benötigt.)</string>
```

Comment:
Better:
... (Wenn diese Option nicht verfügbar ist, so wird die Statusbar durch eine andere Einstellung verwendet)

Reply:
Wold the following description be beter understandabele? Or would it be bether to remove this remark completly? 

EN: If this option cannot be selected, the status bar is forced by another setting.
GER: Wen diese Option nicht ausgewählt werden kann, wird die Statusbar durch eine andere Einstellung benötigt.

Reply:
I personaly would leave them unchanged.

By the way, in the meantime the string translations are moved to crowdin.
Therefore no need to translate here.

Reply:
Removed the braces and it's text.